services/blockchain-indexer/tests/unit/shared/utils/transactions.test.js

```diff
@@ -27,8 +27,8 @@ const {
 } = require('../../../constants/events');
 
 const TRANSACTION_STATUS = Object.freeze({
-	SUCCESS: 'success',
-	FAIL: 'fail',
+	SUCCESSFUL: 'successful',
+	FAILED: 'failed',
 });
```

Comment:
would it be better to import the constant from `services/blockchain-indexer/shared/constants.js` instead?